docs/tutorials/guardrails/use_bedrock_guardrails.md

```diff
@@ -0,0 +1,236 @@
+                      "action": "BLOCKED",
+                      "confidence": "HIGH",
+                      "type": "PROMPT_ATTACK"
+                    }
+                  ]
+                },
+                "wordPolicy": {
+                  "customWords": [
+                    {
+                      "action": "BLOCKED",
+                      "match": "suicide"
+                    }
+                  ]
+                }
+              }
+            ],
+            "blockedResponse": "Sorry, the model cannot answer this question.",
+            "output": [
+              {
+                "text": "Sorry, the model cannot answer this question."
+              }
+            ],
+            "outputs": [
+              {
+                "text": "Sorry, the model cannot answer this question."
+              }
+            ],
+            "usage": {
+              "contentPolicyUnits": 1.0,
+              "contextualGroundingPolicyUnits": 0.0,
+              "sensitiveInformationPolicyFreeUnits": 0.0,
+              "sensitiveInformationPolicyUnits": 0.0,
+              "topicPolicyUnits": 1.0,
+              "wordPolicyUnits": 1.0
+            }
+          }
+        }
+      ],
+      "status_code": 200
+    }
+  ]
+}
+```
+
+# Use the guardrails embedded in the Amazon Bedrock Model Inference API
+
+## 0. Preparation
+Create your own Amazon Bedrock guardrails. For more information, see [Create a guardrail](https://docs.aws.amazon.com/bedrock/latest/userguide/guardrails-create.html).
+
+## 1. Create a connector for an Amazon Bedrock model with guardrail headers
+
+```
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "BedRock claude Connector",
+  "description": "BedRock claude Connector",
+  "version": 1,
+  "protocol": "aws_sigv4",
+  "parameters": {
+      "region": "us-east-1",
+      "service_name": "bedrock",
+      "max_tokens_to_sample": 8000,
+      "temperature": 0.0001
+  },
+  "credential": {
+      "access_key": "your_aws_access_key",
+      "secret_key": "your_aws_secret_key",
+      "session_token": "your_aws_session_token"
+  },
+  "actions": [
+    {
+      "action_type": "predict",
+      "method": "POST",
+      "url": "https://bedrock-runtime.us-east-1.amazonaws.com/model/anthropic.claude-v2/invoke",
+      "headers": { 
+        "content-type": "application/json",
+        "x-amz-content-sha256": "required",
+        "X-Amzn-Bedrock-Trace": "ENABLED",
+        "X-Amzn-Bedrock-GuardrailIdentifier": "your_GuardrailIdentifier",
+        "X-Amzn-Bedrock-GuardrailVersion": "1"
+      },
+      "request_body": "{\"prompt\":\"${parameters.prompt}\", \"max_tokens_to_sample\":${parameters.max_tokens_to_sample}, \"temperature\":${parameters.temperature},  \"anthropic_version\":\"${parameters.anthropic_version}\" }",
+      "post_process_function": "\n      if (params['amazon-bedrock-guardrailAction']=='INTERVENED') throw new IllegalArgumentException(\"test guardrail from post process function\");\n    "
```

Comment:
@jngz-es I think we should enhance this part , like add default process function

Reply:
Yeah, agreed. I can add a default process function in next release version.